PopupMenuDemo/MainActivity.cs

```diff
@@ -0,0 +1,37 @@
+namespace PopupMenuDemo;
+
+[Activity(Label = "PopupMenuDemo", MainLauncher = true)]
+public class Activity1 : Activity
+{
+    protected override void OnCreate(Bundle? bundle)
+    {
+        base.OnCreate(bundle);
+
+        SetContentView(Resource.Layout.Main);
+
+        Button showPopupMenu = FindViewById<Button>(Resource.Id.popupButton)!;
+
+        if (showPopupMenu is null)
+            throw new NullReferenceException();
```

Comment:
Let's approach this with `ArgumentNullException` here for consistency, as we did with the TextSwitcher demo! You can reference the same format here:

 https://github.com/xamarin/monodroid-samples/blob/c5e505e2cc1d723f9c77df9445679ac022a3dbd4/TextSwitcher/TextSwitcher/MainActivity.cs#L18

Reply:
Added ArgumentNullException.ThrowIfNull(showPopupMenu); and removed the if statement.